Costume/100_Voltorb.yaml

```diff
@@ -0,0 +1,20 @@
+species: Voltorb
+imagename: Wooloo
+displayname: Wooloo
+setname: Costume
+tags: [costume]
+item: [Normal Gem]
```

Comment:
This is a gen 5 item and not in PBR

Reply:
Yeah. Closest alternative is Silk Scarf, although it doesn’t have the temporary effect.